javers-core/src/main/java/org/javers/core/graph/LiveGraphFactory.java

```diff
@@ -67,38 +85,6 @@ public static Class getArrayWrapperType() {
         return ArrayWrapper.class;
     }
 
-    private class MapWrapper{
```

Comment:
te klasy celowo były private żeby nie zaśmiecać publicznego API i javadoca, są używane tylko lokalnie, przez LiveGraphFactory. Jeśli chcesz je mieć w oddzielnych plikach to w tym samym pakiecie i package-private a nie public